docs/whatsnew/1.1.rst

```diff
@@ -0,0 +1,182 @@
+.. doctest-skip-all
+
+.. _whatsnew-1.1:
+
+************************
+What's New in SunPy 1.1?
+************************
+
+Overview
+========
+
+The SunPy project is pleased to announce the 1.1 release of the sunpy package.
+
+The headline changes in 1.1 are:
+
+* The `~sunpy.coordinates` subpackage now supports four additional coordinate frames (HCI, HEE, GSE, and GEI).
+* Remote Data Manager (#3124)
+* SUVI support (#3269 / #3301)
+* Initial support for WISPR data (#3501)
+* The import times for `sunpy` and some subpackages are significantly shorter, with no loss of functionality.
```

Comment:
```suggestion
* Significantly shorter import times for `sunpy` and some subpackages, with no loss of functionality.
```

Reply:
I don't really care whether this change is made, but the items in this list should be parallel in construction.  If this change is made, then the first list item should also be a non-sentence.